src/scripts/main.js

```diff
@@ -1,3 +1,20 @@
 'use strict';
 
-// write your code here
+function cloneColumn(section) {
+  for (let i = 0; i < section.rows.length; i++) {
+    const row = section.rows[i];
+    const secondCell = row.cells[1];
+    const lastCell = row.cells[row.cells.length - 1];
+    const clonedCell = secondCell.cloneNode(true);
+
+    row.insertBefore(clonedCell, lastCell);
```

Comment:
It's a good practice to check if the elements you are going to work with actually exist to prevent any potential errors. For instance, before calling `insertBefore` method, you could check if `clonedCell` and `lastCell` exist.